### PR TITLE
feat(logging) Rename logger functions from <Level>b to <Level>

### PR DESCRIPTION
**Pull request by Tetrergeru**

### CODE REVIEW

**Review by kissken — DISMISSED** [review]

**Review by Dimedrolity — APPROVED** [review]

File: senders/twilio/voice.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func (sender *twilioSenderVoice) SendEvents(events moira.NotificationEvents, con
 	if err != nil {
 		return fmt.Errorf("failed to make call to contact %s: %s", contact.Value, err.Error())
 	}
-	sender.logger.Debugb().
+	sender.logger.Debug().
 		String("status", twilioCall.Status).
 		String("callbacl_url", voiceURL).
```
Comment (kissken) [inline senders/twilio/voice.go:28]: String("callbacl_url", voiceURL) oops

File: notifier/selfstate/heartbeat/notifier.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ func GetNotifier(logger moira.Logger, database moira.Database) Heartbeater {
 
 func (check notifier) Check(int64) (int64, bool, error) {
 	if state, _ := check.db.GetNotifierState(); state != moira.SelfStateOK {
-		check.log.Errorb().
+		check.log.Error().
 			String("error", check.GetErrorMessage()).
 			Msg("Send message")
```
Comment (kissken) [inline notifier/selfstate/heartbeat/notifier.go:25]: hmm, seems, here should be NOT send, see first issue in new sentry :)

File: senders/twilio/voice.go (hunk 1 above)
Comment (Tetrergeru) [reply senders/twilio/voice.go]: Fixed

**Review by kissken — APPROVED** [review]

**Review by Dimedrolity — APPROVED** [review]

File: notifier/selfstate/heartbeat/notifier.go (hunk 2 above)
Comment (Tetrergeru) [reply notifier/selfstate/heartbeat/notifier.go]: Fixed

**Pull request merged**